docs/content/docs/guide/resolver.md

```diff
@@ -174,18 +174,51 @@ If the object fetched cannot be parsed as a Tekton `Task` it will error out.
 
 ## Remote Pipeline annotations
 
-Remote Pipeline can be referenced by annotation, this allows you to share your Pipeline definition across.
+Remote Pipeline can be referenced by annotation, allowing you to share a Pipeline across multiple repositories.
 
-Only one Pipeline is allowed in annotation.
+Only one Pipeline is allowed on the `PipelineRun` annotation.
 
-An annotation to a remote pipeline looks like this :
+An annotation to a remote pipeline looks like this, using a remote URL:
 
 ```yaml
 pipelinesascode.tekton.dev/pipeline: "https://git.provider/raw/pipeline.yaml
 ```
 
-It supports remote URL and files inside the same Git repository.
+or from a relative path inside the repository:
 
-{{< hint info >}}
-[Tekton Hub](https://hub.tekton.dev) doesn't currently have support for `Pipeline`.
-{{< /hint >}}
+```yaml
+pipelinesascode.tekton.dev/pipeline: "./tasks/pipeline.yaml
+```
+
+Fetching `Pipelines` from the [Tekton Hub](https://hub.tekton.dev) is not currently supported.
+
+### Overriding tasks from a remote pipeline from a PipelineRun
+
+Remote task annotations on the remote pipeline are supported but no other annotations are supported.
+
+If a user wants to override one of the tasks in the remote pipeline, they can do so by adding a task with the same name
+n their `PipelineRun` annotations.
+
+For example if the user PipelineRun contains those annotations:
+
+```yaml
+kind: PipelineRun
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/pipeline: "https://git.provider/raw/pipeline.yaml
+    pipelinesascode.tekton.dev/task-1: "./my-git-clone-task.yaml
+```
+
+and the Pipeline referenced by the `pipelinesascode.tekton.dev/pipeline` annotation
+in "<https://git.provider/raw/pipeline.yaml>"  contains those annotations:
+
+```yaml
+kind: Pipeline
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/task-1: "./my-task.yaml
+    pipelinesascode.tekton.dev/task-2: "git-clone"
```

Comment:
`task-1`, `task-2` are just "index", they don't "refer" the name of the Task in the pipeline, right ?

Reply:
right, just becasue annotation length are not infinite 🙃